platform.js

```diff
@@ -101,30 +101,34 @@ class App {
 
   copyTemplate () {
     // don't copy the app if the asar filename is specified.
-    if ((this.asarOptions || {}).filename) {
+    if (this.opts.prebuiltAsar) {
+      if (this.opts.afterCopy) {
+        return Promise.reject(new Error('afterCopy is incompatible with prebuiltAsar'))
```

Comment:
You should just be able to `throw new Error`.

Reply:
This isn't an async function, I was trying to make as little of a splash as possible. I hadn't looked to see if the callers setup to handle thrown exceptions and not just promise rejections.